include/SFML/Window/Vulkan.h

```diff
@@ -76,7 +77,9 @@ CSFML_WINDOW_API sfVulkanFunctionPointer sfVulkan_getFunction(const char* name);
 ////////////////////////////////////////////////////////////
 /// \brief Get Vulkan instance extensions required for graphics
 ///
+/// \param count Pointer to a variable that will be filled with the number of modes in the array
```

Comment:
```suggestion
/// \param count Pointer to a variable that will be filled with the number of instance extensions in the array
```

Reply:
I blindly copied that from `sfSoundRecorder_getAvailableDevices`. It's strange, because "modes" is also meaningless there (I think), so maybe it was also blindly copied from elsewhere :smile: